docs/_docs/0.7.0/1_4_powered_by.md

```diff
@@ -146,6 +146,8 @@ Meanwhile, we build a set of data access standards based on Hudi, which provides
 
 21. ["Meetup talk by Nishith Agarwal"](https://www.meetup.com/UberEvents/events/274924537/) - Uber Data Platforms Meetup, Dec 2020
 
+22. ["Apache Hudi Meetup at Uber with talks from AWS, CityStorageSystems & Uber"](https://youtu.be/cAvbBfMbaiA) - By Udit Mehrotra, Wenning Ding (AWS), Alexander Filipchik (CityStorageSystems), Prashant Wason, Satish Kotha (Uber), Feb 2021
```

Comment:
@n3nash the video is private

Reply:
Ack, fixed it, please check again

Reply:
Still private

Reply:
@vburenin : Did you try this https://youtu.be/iXBInMLbjo0. it works for me.